pom.xml

```diff
@@ -90,26 +96,22 @@
             <scope>provided</scope>
         </dependency>
         <dependency>
-            <!-- Baseline of Stapler in 1.625.3, required to fix upper bounds issue in the core -->
             <groupId>org.kohsuke.stapler</groupId>
             <artifactId>stapler</artifactId>
-            <version>1.237</version>
+            <version>1.260</version>
```

Comment:
2.249.1 should include it in the BOM

Reply:
Should I change the jenkins version or the stapler version?